sql/src/main/java/io/crate/executor/transport/task/ShowCreateTableTask.java

```diff
@@ -47,7 +48,7 @@ public void execute(BatchConsumer consumer, Row parameters) {
         Row1 row;
         try {
             CreateTable createTable = MetaDataToASTNodeResolver.resolveCreateTable(tableInfo);
-            row = new Row1(SqlFormatter.formatSql(createTable));
+            row = new Row1(new BytesRef(SqlFormatter.formatSql(createTable)));
```

Comment:
It's kinda related: Now the SizeEstimator for the string type is used which expects BytesRef instead of String.